src/NewsService.Mappers/ResponsesMappers/NewsResponseMapper.cs

```diff
@@ -0,0 +1,30 @@
+﻿using LT.DigitalOffice.Kernel.Exceptions;
+using LT.DigitalOffice.NewsService.Mappers.ResponsesMappers.Interface;
+using LT.DigitalOffice.NewsService.Models.Db;
+using LT.DigitalOffice.NewsService.Models.Dto.Models;
+
+namespace LT.DigitalOffice.NewsService.Mappers.ResponsesMappers
+{
+    public class NewsResponseMapper : INewsResponseMapper
+    {
+        public News Map(DbNews value)
+        {
+            if (value == null)
+            {
+                throw new BadRequestException();
```

Comment:
Maybe you need to use this exc(this is exc from this service): new ArgumentNullException(nameof(request)), but if you don't want to do it, add a message in your exc.